### DISCUSSION TITLE
Difficulty Grouping

### DISCUSSION THREAD

**Discussion by Shibotomoro** [Quaver/Quaver#4783] [Feature Request]
### Is your feature request related to a problem? Please describe.

It's a bit annoying having to scroll through the entire download setlist to find different difficulties, I'd like it so that it's a bit more concisely organized. I understand you can already sort it by difficulties but I'd like it to be more organized so that I can find the songs I want easier.

### Describe the solution you'd like

Make folders for the different difficulty star ratings. Kinda similar to how Osu does it, where all the 1 stars are grouped together and then I can scroll through the precise decimal points. It would be easier than having to scroll through all the 1s then 2s then 3s just so I can get to the 10 stars.  

### Additional context

Something similar to these screenshots?

![image](https://github.com/Quaver/Quaver/assets/55855973/56d1833f-f110-406a-9b0e-8b81db4a2471)
![image](https://github.com/Quaver/Quaver/assets/55855973/f1edc678-ce1a-4958-8d65-f37f4614b07e)

**Comment by Young-TW** [comment]
This featureI’m wholeheartedly in favor of adding difficulty grouping! 

This feature would make navigating the song list so much more efficient—being able to collapse or expand folders by difficulty would save a ton of scrolling and help me jump straight to the exact difficulty I want. Super practical and would greatly improve usability. Can’t wait to see it implemented!
